kannon/master.py

```diff
@@ -146,22 +139,37 @@ def _create_job_object(self, serialized_task: str, job_name: str) -> client.V1Jo
             "--serialized-task",
             f"'{serialized_task}'",
         ]
+        # inherit environment variables
         child_envs = []
         for env_name in self.env_to_inherit:
             if env_name not in os.environ:
                 raise ValueError(f"Envvar {env_name} does not exist.")
             child_envs.append({"name": env_name, "value": os.environ.get(env_name)})
+        # mount persistent volume
+        volume_mounts = []
+        if self.persistent_volume is not None:
+            mount_path = os.environ.get("TASK_WORKSPACE_DIRECTORY", "/cache")
+            volume_mounts.append(client.V1VolumeMount(name=self.persistent_volume, mount_path=mount_path))
         container = client.V1Container(
             name=self.container_name,
             image=self.image_name,
             command=cmd,
             env=child_envs,
+            volume_mounts=volume_mounts,
+            image_pull_policy="IfNotPresent",
```

Comment:
You should not set image_pull_policy for some tags like `latest`.

Reply:
+1. And I think we shouldn't add extra default as much as possible. Was there any reason you needed this?